### PR TITLE
EVEREST-996 autocomplete storybook edit

### PR DESCRIPTION
**Pull request by aElDi**
https://github.com/solovevayaroslavna/everest-students-summer-practice/assets/63113966/994844a1-9224-4fed-89dc-8bb54dfc5ff4

### CODE REVIEW

File: ui/packages/ui-lib/src/form/inputs/auto-complete/auto-complete.stories.tsx [hunk 1]
```diff
@@ -12,8 +13,44 @@ type CustomArgs = React.ComponentProps<typeof AutoCompleteInput> & {
...
+    },
+    toc: true,
+    page: () => (
+      <>
+        <DocBlock.Title />
+        <DocBlock.Subtitle />
+        <DocBlock.Description />
+        <DocBlock.Source
+          code={`
+<AutoCompleteInput
```
Comment (solovevayaroslavna) [inline ui/packages/ui-lib/src/form/inputs/auto-complete/auto-complete.stories.tsx:30]: please run linter with `pnpm format`, in the fork we haven't CI piplines where it works automatically

Comment (solovevayaroslavna) [reply ui/packages/ui-lib/src/form/inputs/auto-complete/auto-complete.stories.tsx:30]: Can I also ask to add some photo/video with PR as we do it in our other PRs?
We do it to simplify the review process, you can check https://github.com/solovevayaroslavna/everest-students-summer-practice/pull/22#issuecomment-2169143098 or https://github.com/solovevayaroslavna/everest-students-summer-practice/pull/25 for examples.

A small demo helps you navigate by what has been done in the work and it's easier to conduct a review. In many teams, this is not necessary, but I strongly recommend that all my students use this approach.

Comment (solovevayaroslavna) [reply ui/packages/ui-lib/src/form/inputs/auto-complete/auto-complete.stories.tsx:30]: all other things looks fine

File: ui/packages/ui-lib/package.json [hunk 2]
```diff
@@ -47,13 +47,15 @@
...
     "@storybook/blocks": "^7.6.6",
+    "@storybook/icons": "^1.2.9",
     "@storybook/react": "^7.6.6",
     "@storybook/react-vite": "^7.6.6",
     "@storybook/test": "^7.6.6",
     "@testing-library/jest-dom": "^6.1.4",
     "@testing-library/react": "^14.0.0",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-storybook": "^0.6.15",
+    "storybook-addon-pseudo-states": "^3.1.1",
```
Comment (solovevayaroslavna) [inline ui/packages/ui-lib/package.json:58]: can we add this one to all our components where is it's needed? I could prepare a separate ticket for it

**Review by solovevayaroslavna — APPROVED** [review]

File: ui/packages/ui-lib/package.json (hunk 2 above)
Comment (aElDi) [reply ui/packages/ui-lib/package.json:58]: Yes, if you want to use `hover` and `focus` pseudo classes in stories with other components, you should add this

File: ui/packages/ui-lib/src/form/inputs/auto-complete/auto-complete.stories.tsx (hunk 1 above)
Comment (aElDi) [reply ui/packages/ui-lib/src/form/inputs/auto-complete/auto-complete.stories.tsx:30]: > please run linter with `pnpm format`, in the fork we haven't CI piplines where it works automatically

this does not make any changes.
I need this formatting to make the code look straight in the storybook

Comment (solovevayaroslavna) [reply ui/packages/ui-lib/src/form/inputs/auto-complete/auto-complete.stories.tsx:30]: [#](https://github.com/solovevayaroslavna/everest-students-summer-practice/pull/27/files#:~:text=need%20this%20formatting%20to%20make%20the)
- yes, you are right) 
but it seems to me that it is customary all over the world to submit PR for review in a formatted version, this makes it easier to read
it is difficult for me as a reviewer to read such code, I will need to go to it locally, run the linter there myself and then view it, I think that cleaning the PR, making it beautiful before showing it, making links to the task and attaching screenshots to it is a simple show of respect for colleagues

**Review by solovevayaroslavna — CHANGES_REQUESTED** [review]
Necessary
- resolve conflicts
- launch the linter

**Pull request closed without merging**